packages/block-editor/src/components/spacing-sizes-control/utils.js

```diff
@@ -181,15 +208,15 @@ export function isValuesMixed( values = {}, sides = ALL_SIDES ) {
  * @return {boolean} Whether values are defined.
  */
 export function isValuesDefined( values ) {
-	return (
-		values !== undefined &&
-		! isEmpty(
-			Object.values( values ).filter(
-				// Switching units when input is empty causes values only
-				// containing units. This gives false positive on mixed values
-				// unless filtered.
-				( value ) => !! value && /\d/.test( value )
-			)
+	if ( values === undefined || values === null ) {
```

Comment:
This change is required because `null` is a valid value for an empty gap value when gap does not use axial sides (this change adds in `null` to the check). Without it, editing the Columns block spacing in the select-list mode threw a fatal error for me:

![image](https://user-images.githubusercontent.com/14988353/186603371-31383d53-31ad-4ca3-a8b6-a5e2595938ea.png)